.storybook/story-layout.js

```diff
@@ -55,15 +55,15 @@ const StoryLayout = ({ children, title, description, source }) => {
               className='z-10 -mb-px'
               variant='lifted'
               value={tab}
-              onChange={(t) => setTab(t)}
+              onChange={(tab) => setTab(tab === 'fullWidthClick' ? 'html' : tab)}
```

Comment:
This may seem weird but this is to preserve the current behaviour while having a different value, therefore a different key in the tab. So React doesn't complain at every component preview.